<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HpiSettingsViewWidget</class>
 <widget class="QWidget" name="HpiSettingsViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HPI Setup</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="m_gridLayout_main">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="m_groupBox_loadDigitizers">
       <property name="minimumSize">
        <size>
         <width>228</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Digitizers</string>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Loaded HPI coils:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="m_label_numberLoadedCoils">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Loaded additional digitizers:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="m_label_numberLoadedDigitizers">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Loaded fiducials:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="m_label_numberLoadedFiducials">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Loaded EEG positions:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="m_label_numberLoadedEEG">
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLineEdit" name="m_lineEdit_filePath">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="m_pushButton_loadDigitizers">
          <property name="text">
           <string>Load Digitizers</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1" rowspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="m_groupBox_hpiFitting">
       <property name="minimumSize">
        <size>
         <width>228</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Fitting</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="17" column="0" colspan="4">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Fitting errors:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="14" column="0" colspan="4">
         <widget class="QToolButton" name="m_pushButton_doSingleFit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Perform single HPI fit</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" rowspan="2" colspan="3">
         <widget class="QTableWidget" name="tableWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>114</height>
           </size>
          </property>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>30</number>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>80</number>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>22</number>
          </attribute>
          <row>
           <property name="text">
            <string>0</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Coil #</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Frequency</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>155</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>165</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string>190</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string>200</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="8" column="0" colspan="4">
         <widget class="QCheckBox" name="m_checkBox_useComp">
          <property name="text">
           <string>Use compensators</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="m_pushButton_removeCoil">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="18" column="0" colspan="4">
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>113</height>
           </size>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>22</number>
          </attribute>
          <row>
           <property name="text">
            <string>0</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>1</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>2</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>3</string>
           </property>
          </row>
          <column>
           <property name="text">
            <string>Coil #</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Error in mm</string>
           </property>
          </column>
          <item row="0" column="0">
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item row="0" column="1">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="1" column="0">
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item row="1" column="1">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="2" column="0">
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item row="2" column="1">
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item row="3" column="0">
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item row="3" column="1">
           <property name="text">
            <string>0</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="12" column="0" colspan="4">
         <widget class="QCheckBox" name="m_checkBox_continousHPI">
          <property name="text">
           <string>Do continous HPI fitting</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="4">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Coil frequencies:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="m_pushButton_addCoil">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="4">
         <widget class="QCheckBox" name="m_checkBox_useSSP">
          <property name="text">
           <string>Use SSP</string>
          </property>
         </widget>
        </item>
        <item row="21" column="0" colspan="4">
         <widget class="QLabel" name="m_label_fitFeedback">
          <property name="text">
           <string>No fit yet</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="15" column="0" colspan="4">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="4">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="20" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Error threshold:</string>
          </property>
         </widget>
        </item>
        <item row="20" column="2">
         <widget class="QDoubleSpinBox" name="m_doubleSpinBox_maxHPIContinousDist">
          <property name="maximum">
           <double>9999.989999999999782</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="19" column="2">
         <widget class="QLabel" name="m_label_averagedFitError">
          <property name="text">
           <string>0mm</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="19" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Average:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
